roles/openshift_clock/meta/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+galaxy_info:
+  author: Jeremiah Stuever
+  description: OpenShift Clock
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 1.9
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  categories:
+  - cloud
+dependencies:
+- { role: openshift_clock_facts }
```

Comment:
Since there is no longer an upstream role, I think it might make sense to consolidate openshift_clock_facts and openshift_clock.